csds-client/client/test_util.go

```diff
@@ -0,0 +1,72 @@
+package client
```

Comment:
Should this be some kind of clientutil package?

Reply:
Moved util.go and test_util.go to the package client/util